src/Moryx.AbstractionLayer.Products.Endpoints/ProductConverter.cs

```diff
@@ -46,7 +46,9 @@ public RecipeDefinitionModel ConvertRecipeType(Type recipeType)
                 HasWorkplans = typeof(IWorkplanRecipe).IsAssignableFrom(recipeType)
             };
         }
-        public ProductModel ConvertProduct(IProductType productType, bool flat)
+
+        /// <param name="includeParts">Determines whether parts should be included</param>
+        public ProductModel ConvertProduct(IProductType productType, bool flat, bool includeParts = true)
```

Comment:
The only difference I see between `bool flat` (meaning no properties, recipes or parts) and the new argument is to load 2/3 of the default page but not the part links. This also would have to be a new method as it would break a public API.